flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "skip-icons-check": true,
+    "only-arches": ["x86_64"],
+    "automerge-flathubbot-prs": true
```

Comment:
Just to make sure - this will make the Flathub bots automatically merge and deploy new releases of your plugin. Are you okay with this?

Auto-merge is very convenient, but some developers prefer the bots to only open the pull request, so they can test first and merge manually after that. For Flathub, either way is fine, developers choose what fits their workflow better. Anyway, just double checking here.

Reply:
Yep - this works for us.